qa/workunits/ceph-helpers.sh

```diff
@@ -206,6 +217,8 @@ function test_kill_daemon() {
         kill_daemon $pidfile TERM || return 1
     done
 
+    # Wait a bit to get things reported down.
+    sleep 3
```

Comment:
If you ever run into this again, please ping me about it. There is absolutely no reason why timing should be an issue here.